src/screens/qr/components/BaseQRCodeScreen.tsx

```diff
@@ -0,0 +1,56 @@
+import React, {useCallback, useContext} from 'react';
+import {useNavigation} from '@react-navigation/native';
+import {KeyboardAvoidingView, Platform, StyleSheet, ScrollView} from 'react-native';
+import {SafeAreaView} from 'react-native-safe-area-context';
+import {Box} from 'components';
+import {Toolbar} from './Toolbar';
+import {useI18n} from 'locale';
+
+interface BaseQRCodeScreenProps {
+  children?: React.ReactNode;
+  showBackButton?: boolean;
+  showCloseButton?: boolean;
+}
+
+export const BaseQRCodeScreen = ({children, showBackButton, showCloseButton}: BaseQRCodeScreenProps) => {
+  const navigation = useNavigation();
+  const i18n = useI18n();
+  const close = useCallback(() => navigation.navigate('Home'), [navigation]);
+  return (
+    <KeyboardAvoidingView style={styles.flex} behavior={Platform.OS === 'ios' ? 'padding' : 'height'}>
```

Comment:
I don't think we need the `KeyboardAvoidingView` here. The purpose of this is to move things in the View when the device keyboard pops up. We're using this on the DataSharing flow because the OTK entry page, but there is nothing  in the QR Code flow that pops the keyboard.